src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -33,26 +33,21 @@ abstract class AbstractFunctionalTest extends Specification {
     def setup() {
         projectDir = temporaryFolder.root
         buildFile = temporaryFolder.newFile('build.gradle')
-
-        def pluginClasspathResource = getClass().classLoader.findResource("plugin-classpath.txt")
-        if (pluginClasspathResource == null) {
-            throw new IllegalStateException("Did not find plugin classpath resource, run 'functionalTestClasses' build task.")
-        }
-
-        def pluginClasspath = pluginClasspathResource.readLines()
-                .collect { it.replace('\\', '\\\\') } // escape backslashes in Windows paths
-                .collect { "'$it'" }
-                .join(", ")
-
-        // Add the logic under test to the test build
+                
+        def pluginClasspath = GradleRunner.create().withPluginClasspath().getPluginClasspath()
+            .collect { it.path.replace('\\', '\\\\') } // escape backslashes in Windows paths
```

Comment:
Oh, I see what's wrong here. The buildscript shouldn't use the `buildscript` notation anymore. Instead it should use the plugin DSL. See this [commit](https://github.com/bmuschko/gradle-tomcat-plugin/commit/bacc2dc9ad5695f67b4b9f5b4e6509c05c3f8def) for more info.